client/app/scripts/superdesk-authoring/views/header-info.html

```diff
@@ -0,0 +1,136 @@
+<button class="proofread-toggle btn btn-light" ng-click="proofread = !proofread">
+    <span class="button-lable" translate>Mode: </span>
+    <span ng-show="proofread" translate>Proofread</span>
+    <span ng-show="!proofread" translate>Normal</span>
+</button>
+
+<span sd-theme-select ng-if="proofread" data-key="proofreadTheme"></span>
+<span sd-theme-select ng-if="!proofread" data-key="theme"></span>
+
+<ul class="metadata-list" ng-controller="MetadataWidgetCtrl" ng-if="loaded">
+    <li class="date-info full-width clearfix">
+        <label translate>Created&nbsp;</label>

```

Comment:
Yeah, but if you change font size you need to change padding-right also... &nbsp; adjusts to font-size! But yes, bigger problem is for translating! :+1: